Makefile

```diff
@@ -19,6 +19,18 @@ build:
 	# Go code.
 	go build -o out/x86/basic-go src/main.go
 
+	gcc src/basic-cpp.cpp -o out/arm64/basic-cpp -g
+	gcc src/basic-cpp-plt.cpp -o out/arm64/basic-cpp-plt
+	gcc src/basic-cpp.cpp -o out/arm64/basic-cpp-no-fp -fomit-frame-pointer
+	gcc src/basic-cpp.cpp -o out/arm64/basic-cpp-no-fp-with-debuginfo -fomit-frame-pointer -g
+	gcc src/basic-cpp-plt.cpp -o out/arm64/basic-cpp-plt-pie -pie -fPIE
+	gcc src/basic-cpp-plt.cpp -o out/arm64/basic-cpp-plt-hardened -pie -fPIE -fstack-protector-all -D_FORTIFY_SOURCE=2 -Wl,-z,now -Wl,-z,relro -O2
+	# The JIT code has frame pointers.
+	gcc src/basic-cpp-jit.cpp -o out/arm64/basic-cpp-jit -g
```

Comment:
The toy jit only emits x86 code, let's remove this meanwhile and later on we can adapt it to generate arm64 jit code, too